cookbooks/aws-parallelcluster-environment/files/default/ec2_udev_rules/ec2_dev_2_volid.py

```diff
@@ -132,5 +98,54 @@ def main():
     print(volume_id)
 
 
+def main():
+    syslog.syslog("Starting ec2_dev_2_volid.py script")
+    # Get dev
+    try:
+        dev = str(sys.argv[1])
+        validate_device_name(dev)
+        syslog.syslog(f"Input block device is {dev}")
+    except IndexError:
+        syslog.syslog(syslog.LOG_ERR, "Provide block device i.e. xvdf")
+
+    # Convert dev to mapping format
+    dev = convert_dev(dev)
+
+    # Get IMDSv2 token
```

Comment:
Minor: You used the right self-explicative name for the function so you can avoid the comment.

This kind of comments are useless since in the comment you're writing the name of the function you're calling "get_imdsv2_token".